bpf/failsafes/failsafes.go

```diff
@@ -95,9 +97,33 @@ func (m *Manager) ResyncFailsafes() error {
 			log.WithField("proto", p.Protocol).Warn("Ignoring failsafe port; protocol not supported in BPF mode.")
 			return
 		}
-		k := MakeKey(ipProto, p.Port, outbound)
+
+		// Slice up the CIDR and convert the string representations to uints
+		parts := strings.Split(p.Net, "/")
+		ipStr := parts[0]
+		maskStr := parts[1]
```

Comment:
The FV tests fail because there is no parts[1], need to be handled as a special case if there is no mask.
```
felix-1-672602-7-felixfv[stderr] panic: runtime error: index out of range [1] with length 1
felix-1-672602-7-felixfv[stderr] 
felix-1-672602-7-felixfv[stderr] goroutine 30 [running]:
felix-1-672602-7-felixfv[stderr] panic(0x32a20a0, 0xc000d89500)
felix-1-672602-7-felixfv[stderr]        /usr/local/go-cgo/src/runtime/panic.go:1064 +0x470 fp=0xc000a6f1e0 sp=0xc000a6f128 pc=0x449030
felix-1-672602-7-felixfv[stderr] runtime.goPanicIndex(0x1, 0x1)
felix-1-672602-7-felixfv[stderr]        /usr/local/go-cgo/src/runtime/panic.go:88 +0xa5 fp=0xc000a6f228 sp=0xc000a6f1e0 pc=0x446cc5
felix-1-672602-7-felixfv[stderr] github.com/projectcalico/felix/bpf/failsafes.(*Manager).ResyncFailsafes.func2(0x0, 0x0, 0x314e948, 0x3, 0x16, 0xc0009cce00)
felix-1-672602-7-felixfv[stderr]        /go/src/github.com/projectcalico/felix/bpf/failsafes/failsafes.go:104 +0x8c7 fp=0xc000a6f3d0 sp=0xc000a6f228 pc=0x2b072c7
felix-1-672602-7-felixfv[stderr] github.com/projectcalico/felix/bpf/failsafes.(*Manager).ResyncFailsafes(0xc00041e6c0, 0x0, 0x0)
felix-1-672602-7-felixfv[stderr]        /go/src/github.com/projectcalico/felix/bpf/failsafes/failsafes.go:134 +0x4b8 fp=0xc000a6f528 sp=0xc000a6f3d0 pc=0x2b05d18
felix-1-672602-7-felixfv[stderr] github.com/projectcalico/felix/bpf/failsafes.(*Manager).CompleteDeferredWork(0xc00041e6c0, 0x34288a7, 0xd)
felix-1-672602-7-felixfv[stderr]        /go/src/github.com/projectcalico/felix/bpf/failsafes/failsafes.go:65 +0x70 fp=0xc000a6f560 sp=0xc000a6f528 pc=0x2b05810
felix-1-672602-7-felixfv[stderr] github.com/projectcalico/felix/dataplane/linux.(*InternalDataplane).apply(0xc0009d8b00)
felix-1-672602-7-felixfv[stderr]        /go/src/github.com/projectcalico/felix/dataplane/linux/int_dataplane.go:1669 +0x1d9 fp=0xc000a6fb20 sp=0xc000a6f560 pc=0x2d16459
felix-1-672602-7-felixfv[stderr] github.com/projectcalico/felix/dataplane/linux.(*InternalDataplane).loopUpdatingDataplane(0xc0009d8b00)
felix-1-672602-7-felixfv[stderr]        /go/src/github.com/projectcalico/felix/dataplane/linux/int_dataplane.go:1563 +0x8bc fp=0xc000a6ffd8 sp=0xc000a6fb20 pc=0x2d13fbc
felix-1-672602-7-felixfv[stderr] runtime.goexit()
felix-1-672602-7-felixfv[stderr]        /usr/local/go-cgo/src/runtime/asm_amd64.s:1374 +0x1 fp=0xc000a6ffe0 sp=0xc000a6ffd8 pc=0x4816c1
felix-1-672602-7-felixfv[stderr] created by github.com/projectcalico/felix/dataplane/linux.(*InternalDataplane).Start
felix-1-672602-7-felixfv[stderr]        /go/src/github.com/projectcalico/felix/dataplane/linux/int_dataplane.go:990 +0x5f

```